api/types/accesslist/accesslist.go

```diff
@@ -258,6 +261,12 @@ type Grants struct {
 	Traits trait.Traits `json:"traits" yaml:"traits"`
 }
 
+// AccessListStatus contains dynamic fields calculated during retrieval.
+type AccessListStatus struct {
```

Comment:
`accesslist.AccessListStatus` stutters, I'd keep the Go-side struct as `Status` like `Spec`.